src/entity.js

```diff
@@ -0,0 +1,51 @@
+import { _Entity } from "ecsy";
+import {
+  Object3DComponent,
+  MeshTagComponent,
+  SceneTagComponent,
+  CameraTagComponent
+} from "./components/index.js";
+
+function tagClassForObject3D(obj) {
```

Comment:
Register inflaters for adding tagsComponents for Object3D. `ecsyThreeWorld.registerObject3DTagComponentsInflaters`